fastlane/lib/fastlane/actions/push_git_tags.rb

```diff
@@ -5,15 +5,13 @@ def self.run(params)
         command = [
           'git',
           'push',
-          '--tags'
+          params[:remote],
```

Comment:
Is it an issue that this is potentially `nil`? Until now we only added the `remote` to the array if there is actually a value

Reply:
I just followed `push_to_git_remote`'s spec to make this logic simpler, but totally fine with keeping it optional (=not providing the default value) if you want!

https://github.com/fastlane/fastlane/blob/master/fastlane/lib/fastlane/actions/push_to_git_remote.rb#L61

Reply:
I think somebody could set `params[:remote]` to nil, and that would override the default, but that being said, if you do that, you deserve the crash 😆